examples/gkeenv.list

```diff
@@ -0,0 +1,13 @@
+PROJECT_ID=testing-project-id
+NAME_PREFIX=test-aiinfra-gke
+ZONE=us-central1-a
+ACCELERATOR_TYPE=nvidia-tesla-a100
+METADATA={ meta1 = "val", meta2 = "val2" }
+LABELS={ label1 = "marker1" }
+ORCHESTRATOR_TYPE=gke
+INSTANCE_COUNT=1
+NETWORK_CONFIG=default_network
+GPU_COUNT=2
+GKE_NODE_POOL_COUNT=2
```

Comment:
How do these 3 params below co-exist with each other? Does the last one takes precedence if present, and makes the first 2 unnecessary?

Reply:
Actually the contention is between `GKE_NODE_COUNT` and `INSTANCE_COUNT`. Everything else is needed whether GKE or not.
The `GKE_NODE_COUNT` takes effect only if `ORCHESTRATOR_TYPE=gke` and takes precedence over  `INSTANCE_COUNT`.

Reply:
renamed `GKE_NODE_COUNT` to `GKE_NODE_COUNT_PER_NODE_POOL` and added validations as discussed.